### ISSUE TITLE
FR: Add {{dateIso}}

### ISSUE THREAD

**Issue by madduck** [eikek/docspell#2265]
This is related to #577 but not fully the same, I don't think.

Please provide `{{dateIso}}` and maybe a switch to ignore locale for dates, and just to use ISO instead?

**Comment by eikek** [comment]
I'm not really sure what that applies to, do you mean the ui templates that are rendered in an item card?

**Comment by madduck** [comment]
Yes, there, but I'd also think that the same code could be accessed for https://github.com/docspell/dsc/issues/114 and #2270
